src/secp256k1/src/secp256k1.c

```diff
@@ -360,6 +360,7 @@ int secp256k1_ecdsa_sign(const secp256k1_context* ctx, secp256k1_ecdsa_signature
     secp256k1_scalar sec, non, msg;
     int ret = 0;
     int overflow = 0;
+    const unsigned char secp256k1_ecdsa_der_algo16[17] = "ECDSA+DER       ";
```

Comment:
same here, given that 17 shows up again, I wonder if you're missing a right padding space in the string
```
>>> len("ECDSA+DER       ")
16
```